includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -184,19 +190,59 @@ public function run() {
 
 		// And make the edit
 		$toSave = $this->simplePutStrategy->apply( $reconciliationItem->getItem(), $inputEntity );
+
 		$editEntity = $this->editEntityFactory->newEditEntity(
 			// TODO use a real user
 			\User::newSystemUser( 'WikibaseReconcileEditReconciliator' ),
 			$toSave->getId(),
 			$reconciliationItem->getRevision()
 		);
-		$saveStatus = $editEntity->attemptSave(
-			$toSave,
-			'Reconciliation Edit',
-			$reconciliationItem->getRevision() ? null : EDIT_NEW,
-			// TODO actually do a token check?
-			false
-		);
+
+		$saveStatus = Status::newGood();
+		foreach ( $otherItems as $otherItem ) {
+			if ( !( $otherItem instanceof ReconciliationItem ) ) {
```

Comment:
Can this happen? The other items should always be reconciliation items, no?

Reply:
done